packages/liveblocks-react/src/factory.tsx

```diff
@@ -214,6 +215,13 @@ export function createRoomContext<
     return room;
   }
 
+  function useStatus(): Status {
+    const room = useRoom();
+    const subscribe = room.events.status.subscribe;
+    const getSnapshot = room.getStatus;
+    return useSyncExternalStore(subscribe, getSnapshot, getSnapshot);
```

Comment:
Your new favourite API! So simple here 👍

Reply:
It's so good indeed! 💯